src/gui/test/app/__snapshots__/layout.tsx.snap

```diff
@@ -0,0 +1,94 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`renders Layout for the "/dashboard" view 1`] = `
+
+<div class="flex flex-col min-h-svh w-full">
+  <div class="flex flex-row min-h-svh w-full">
+    <gui-nav-sidebar>
+    </gui-nav-sidebar>
+    <div class="flex flex-col grow w-full">
+      <gui-nav-header>
+      </gui-nav-header>
+      <gui-dashboard>
+      </gui-dashboard>
+      <gui-nav-footer>
+      </gui-nav-footer>
+    </div>
+  </div>
+  <div
+    role="region"
+    aria-label="Notifications (F8)"
+    tabindex="-1"
+    style="pointer-events: none;"
+  >
+    <ol
+      tabindex="-1"
+      class="fixed top-0 z-[100] flex max-h-screen w-full flex-col-reverse p-4 sm:bottom-0 sm:right-0 sm:top-auto sm:flex-col md:max-w-[420px]"
+    >
+    </ol>
+  </div>
+</div>
+
+`;
+
+exports[`renders Layout for the "/error" view 1`] = `
+
+<div class="flex flex-col min-h-svh w-full">
+  <div class="flex flex-row min-h-svh w-full">
+    <gui-nav-sidebar>
+    </gui-nav-sidebar>
+    <div class="flex flex-col grow w-full">
+      <gui-nav-header>
+      </gui-nav-header>
+      <gui-error-found>
+      </gui-error-found>
+      <gui-nav-footer>
+      </gui-nav-footer>
+    </div>
+  </div>
+  <div
+    role="region"
+    aria-label="Notifications (F8)"
+    tabindex="-1"
+    style="pointer-events: none;"
+  >
+    <ol
+      tabindex="-1"
+      class="fixed top-0 z-[100] flex max-h-screen w-full flex-col-reverse p-4 sm:bottom-0 sm:right-0 sm:top-auto sm:flex-col md:max-w-[420px]"
+    >
+    </ol>
+  </div>
+</div>
+
+`;
+
+exports[`renders Layout for the "/explore" view 1`] = `
+
+<div class="flex flex-col min-h-svh w-full">
+  <div class="flex flex-row min-h-svh w-full">
+    <gui-nav-sidebar>
+    </gui-nav-sidebar>
+    <div class="flex flex-col grow w-full">
+      <gui-nav-header>
+      </gui-nav-header>
+      <gui-explorer>
+      </gui-explorer>
+      <gui-nav-footer>
+      </gui-nav-footer>
+    </div>
+  </div>
+  <div
+    role="region"
+    aria-label="Notifications (F8)"
+    tabindex="-1"
+    style="pointer-events: none;"
+  >
+    <ol
+      tabindex="-1"
+      class="fixed top-0 z-[100] flex max-h-screen w-full flex-col-reverse p-4 sm:bottom-0 sm:right-0 sm:top-auto sm:flex-col md:max-w-[420px]"
+    >
+    </ol>
+  </div>
```

Comment:
I see that this is repeated in many snapshots, is this intentional?

Reply:
Looks like this is the `<Toaster />`'s `ToastViewport` component from `components/ui/toast.tsx`. I've added the mock for it in the `layout.tsx` test!

Reply:
ooohhh I see 👍 makes sense!